example-application.properties

```diff
@@ -0,0 +1,11 @@
+# This file should be copied and filled in to provide instance-specific configuration
+
```

Comment:
server.port=8081

Reply:
Not required. If you're not already running Tomcat (or anything else on port 8080) then the default is fine. For production, we'll most likely put NGINX in front of it, so the default port is fine.